src/layman/upgrade/upgrade_v2_0.py

```diff
@@ -22,6 +22,21 @@ def adjust_db_for_description():
     db_util.run_statement(statement)
 
 
+def adjust_db_for_map_layer_relation():
+    logger.info(f'    Alter DB prime schema map_layer table for UUID')
+
+    statement = f'''
+    ALTER TABLE {DB_SCHEMA}.map_layer ADD COLUMN IF NOT EXISTS
+    layer_uuid  uuid default null;'''
+    db_util.run_statement(statement)
+    statement = f'''
+    ALTER TABLE {DB_SCHEMA}.map_layer ALTER COLUMN layer_workspace DROP NOT NULL;'''
+    db_util.run_statement(statement)
+    statement = f'''
+    ALTER TABLE {DB_SCHEMA}.map_layer ALTER COLUMN layer_name DROP NOT NULL;'''
+    db_util.run_statement(statement)
```

Comment:
Just to clarify expectations: I expect we will drop columns layer_workspace and layer_name later in this milestone.